phalcon/Mvc/Collection.zep

```diff
@@ -1194,24 +1205,18 @@ abstract class Collection implements EntityInterface, CollectionInterface, Injec
          * We always use safe stores to get the success state
          * Save the document
          */
-        let status = collection->update(
+        let status = collection->updateOne(
             [
                 "_id": this->_id
             ],
-            data,
+            ["$set": data],
             [
                 "w": true
             ]
         );
 
-        if typeof status == "array" {
-            if fetch ok, status["ok"] {
-                if ok {
-                    let success = true;
-                }
-            }
-        } else {
-            let success = false;
+        if status->isAcknowledged() {
+            let success = true;
```

Comment:
Otherwise NULL?

Reply:
What do you mean?

Reply:
I mean that `success` is NULL before this line

Reply:
I made some modifications, is that good for you?